GA-worksheet.md

```diff
@@ -7,62 +7,62 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of a header tag is for headers.
```

Comment:
This is correct, but a little vague! The header tag actually holds the introductory material for the website, such as a banner, logo, or navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify). It sits at the top of the body.